backend/courses/management/commands/form_simple_topics.py

```diff
@@ -0,0 +1,81 @@
+from django.core.management.base import BaseCommand
+from django.db import transaction
+from django.db.models import Exists, F, OuterRef
+
+from alert.management.commands.recomputestats import garbage_collect_topics
+from courses.management.commands.load_crosswalk import get_crosswalk_s3
+from courses.models import Course, Topic
+
+
+def form_simple_topics():
+    Course.objects.all().update(topic=None)
+    garbage_collect_topics()
+
+    print("Cleared topics.")
```

Comment:
Should we have these print statements only run `if verbose`?

Reply:
This is a good point but I broke my dev env and I'm afraid of making any changes without testing locally :( will add it to my to do list though.